src/docs/product/alerts/create-alerts/index.mdx

```diff
@@ -12,7 +12,6 @@ The minimum role required to create alerts is member. Sentry users with manager
 
 To create alerts:
 1. Navigate to **Alerts** and click "Create Alert Rule".
-1. Select your project.
 1. Choose what you want to be alerted about. Selecting “Issues” creates an [issue alert](/product/alerts/alert-types/#issue-alerts), while selecting any other option creates a [metric alert](/product/alerts/alert-types/#metric-alerts).
     ![Choice between alerts about Errors, Sessions, Performance, or a Custom Metric](create-new-alert-rule.png)
```

Comment:
This screenshot also needs to be updated since it shows the project as already selected.

Reply:
👍